### PR TITLE
Note the default Ktlint style sections in docs

### PR DESCRIPTION
**Pull request by Goooler**
Closes #2090.

### CODE REVIEW

File: plugin-gradle/README.md [hunk 1]
```diff
@@ -418,6 +418,8 @@ spotless {
       .editorConfigOverride(
         mapOf(
           "indent_size" to 2,
+          // intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles.
+          "ktlint_code_style" to "intellij_idea",
```
Comment (Jaehwa-Noh) [inline plugin-gradle/README.md:422]: You says this is default, but why default override to default?

File: plugin-gradle/README.md [hunk 2]
```diff
@@ -418,6 +418,8 @@ spotless {
       .editorConfigOverride(
         mapOf(
           "indent_size" to 2,
+          // intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles.
```
Comment (Jaehwa-Noh) [inline plugin-gradle/README.md:421]:
```suggestion
          // intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles.
          //this is the example of overriding to `android_studio` style.
```

File: plugin-gradle/README.md (hunk 1 above)
Comment (Jaehwa-Noh) [inline plugin-gradle/README.md:422]:
```suggestion
          "ktlint_code_style" to "android_studio",
```

File: plugin-maven/README.md [hunk 3]
```diff
@@ -419,6 +419,8 @@ Additionally, `editorConfigOverride` options will override what's supplied in `.
   <editorConfigOverride> <!-- optional -->
     <ij_kotlin_allow_trailing_comma>true</ij_kotlin_allow_trailing_comma>
     <ij_kotlin_allow_trailing_comma_on_call_site>true</ij_kotlin_allow_trailing_comma_on_call_site>
+    <!-- intellij_idea is the default style we preset in Spotless, you can override it referring to https://pinterest.github.io/ktlint/latest/rules/code-styles. -->
+    <ktlint_code_style>intellij_idea</ktlint_code_style>
```
Comment (Jaehwa-Noh) [inline plugin-maven/README.md:423]:
```suggestion
    <ktlint_code_style>android_studio</ktlint_code_style>
```

**Comment by Goooler** [comment]
I wrote `intellij_idea` there cause it's an optional flag.

File: plugin-gradle/README.md (hunk 2 above)
Comment (Goooler) [reply plugin-gradle/README.md:421]: One more thing, a blank should be asserted after `//`.

**Pull request merged**